ui/src/app/admin/_components/personTableTooltip.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import React, { useRef } from 'react';
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger, TooltipArrow } from '@/components/ui/tooltip';
+import { Side } from '@floating-ui/utils';
+
+const PersonTableTooltip = ({
+    desc,
+    value,
+    side
+}: {
+    desc: React.ReactNode;
+    value: string;
+    side: Side | undefined;
```

Comment:
I think the type for side is supposed to be a string or more precisely: "top" | "right" | "bottom" | "left"